dmaws/context.py

```diff
@@ -19,6 +20,18 @@ def __init__(self, home=os.getcwd(), dry_run=False):
         self.stacks = {}
         self.create_dependencies = False
 
+    def new_context(self, stage, environment, vars_files):
+        """Create a new context with different variables"""
+        ctx = Context()
+        ctx.stage = stage
+        ctx.environment = environment
+        ctx.load_variables(
+            files=ctx.get_variables_files(True, vars_files))
+        ctx.stacks = copy.deepcopy(self.stacks)
```

Comment:
We can store the `stacks_file` in `ctx.stacks_file`, since we'd still need to find a way to pass it in even if we move the logic from `@cli_command`. And most commands still need one context, so I'm not sure if moving it out is worth it right now.